modules/serverless-install-web-console.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * /serverless/installing_serverless/installing-openshift-serverless.adoc
+
+[id="serverless-install-web-console_{context}"]
+= Installing the {ServerlessOperatorName}
+
+This procedure describes how to install and subscribe to the {ServerlessOperatorName} from the OperatorHub using the {product-title} web console.
+
+.Procedure
+
+. Navigate in the web console to the *Catalog* → *OperatorHub* page.
+. Scroll, or type they keyword *Serverless* into the *Filter by keyword box* to find the {ServerlessOperatorName}.
++
+image::serverless-search.png[{ServerlessOperatorName} in the {product-title} web console]
+
+. Read the information about the Operator and click *Install*.
++
+image::serverless-operator.png[{ServerlessOperatorName} information]
+
+. On the *Create Operator Subscription* page:
++
+image::serverless-create-sub.png[Create Operator Subscription page]
+
+.. The *Installation Mode* is *All namespaces on the cluster (default)*. This mode  installs the Operator in the default `openshift-operators` namespace to watch and be made available to all namespaces in the cluster.
+.. The *Installed Namespace* will be `openshift-operators`.
+.. Select an *Update Channel*.
+... The *4.3* channel will enable installation of the latest stable release of the {ServerlessOperatorName}.
+... The *preview-4.3* channel enables installation of the latest preview version of the {ServerlessOperatorName}, which may contain features that are not yet available from the *4.3* update channel.
+.. Select *Automatic* or *Manual* approval strategy.
+. Click *Subscribe* to make the Operator available to the selected namespaces on this {product-title} cluster.
+. From the *Catalog* → *Operator Management* page, you can monitor the {ServerlessOperatorName} Subscription’s installation and upgrade progress.
+.. If you selected a *Manual* approval strategy, the Subscription’s upgrade status will remain *Upgrading* until you review and approve its Install Plan. After approving on the *Install Plan* page, the Subscription upgrade status moves to *Up to date*.
```

Comment:
```suggestion
.. If you selected a *Manual* approval strategy, the Subscription’s upgrade status will remain *Upgrading* until you review and approve its *Install Plan*. After approving on the *Install Plan* page, the Subscription upgrade status moves to *Up to date*.
```

Reply:
I don't think this really needs that kind of emphasis because it's referring this time to an object as opposed to the page name?
e.g. Subscription doesn't get *strong* text either :) actually subscription and install plan should probably be lower case in these instances.

@jboxman care to pitch in? I know you were talking about capitalization of terms etc recently in a call :slightly_smiling_face: